blocks/form/form.js

```diff
@@ -0,0 +1,165 @@
+import buildForm from '../../utils/forms/forms.js';
+
+const fields = [
+  {
+    type: 'input',
+    label: 'First Name',
+    name: 'firstName',
+    placeholder: 'Enter your first name',
+    required: true,
+    // This validation should have only the extras, if any,
+    // to be added to a field that aren't already on the field itself
+    validation: ['no-nums'],
+  },
+  {
+    type: 'email',
+    label: 'Email',
+    name: 'email',
+    placeholder: 'Enter your email',
+    required: true,
+  },
+  {
+    type: 'tel',
+    label: 'Phone Number',
+    name: 'phone',
+    placeholder: '000-000-0000',
+    required: true,
+    validation: ['phone:US'],
+  },
+  {
+    type: 'textarea',
+    label: 'How did you hear about us?',
+    name: 'referralSource',
+    placeholder: 'e.g., friend, social media, etc.',
+    rows: 8,
+    required: true,
+  },
+  {
+    type: 'number',
+    label: 'Quantity',
+    name: 'quantity',
+    min: 0,
+    max: 100,
+    value: 0, // default value
+    required: true,
+  },
+  {
+    type: 'radio',
+    label: 'Select a flavor',
+    name: 'flavor',
+    options: [
+      { label: 'Vanilla', value: 'vanilla' },
+      { label: 'Chocolate', value: 'chocolate' },
+      { label: 'Strawberry', value: 'strawberry' },
```

Comment:
it might be worth adding a default value when creating the radio, setting the value to label.toLowerCase() or toClassName(label), _unless_ otherwise indicated